Postman Test Suite/Ed-Fi ODS-API Multiple Authorization Strategy Test Suite.postman_collection.json

```diff
@@ -2037,65 +2037,6 @@
 										}
 									},
 									"response": []
-								},
-								{
-									"name": "Get StudentSectionAttendanceEvents",
```

Comment:
This test on this resource wasn't actually testing anything.

The _/ed-fi/studentSectionAssociations_ resource does have special security metadata configuration applying multiple authorization strategies. Here's the declarative representation of this security metadata:

```xml
<Claim name="http://ed-fi.org/ods/identity/claims/ed-fi/studentSectionAssociation">
  <ClaimSets>
    <ClaimSet name="Multiple Auth Test">
      <Actions>
        <Action name="Create">
          <AuthorizationStrategyOverrides>
            <AuthorizationStrategy name="RelationshipsWithEdOrgsAndPeople" />
            <AuthorizationStrategy name="RelationshipsWithStudentsOnlyThroughResponsibility" />
          </AuthorizationStrategyOverrides>
        </Action>
        <Action name="Read">
          <AuthorizationStrategyOverrides>
            <AuthorizationStrategy name="OwnershipBased" />
            <AuthorizationStrategy name="RelationshipsWithEdOrgsAndPeople" />
            <AuthorizationStrategy name="RelationshipsWithStudentsOnlyThroughResponsibility" />
          </AuthorizationStrategyOverrides>
        </Action>
        <Action name="Update">
          <AuthorizationStrategyOverrides>
            <AuthorizationStrategy name="OwnershipBased" />
            <AuthorizationStrategy name="RelationshipsWithEdOrgsAndPeople" />
            <AuthorizationStrategy name="RelationshipsWithStudentsOnlyThroughResponsibility" />
          </AuthorizationStrategyOverrides>
        </Action>
        <Action name="Delete">
          <AuthorizationStrategyOverrides>
            <AuthorizationStrategy name="OwnershipBased" />
            <AuthorizationStrategy name="RelationshipsWithEdOrgsAndPeople" />
            <AuthorizationStrategy name="RelationshipsWithStudentsOnlyThroughResponsibility" />
          </AuthorizationStrategyOverrides>
        </Action>
      </Actions>
    </ClaimSet>
  </ClaimSets>
</Claim>
```
However, the _/ed-fi/studentSectionAttendanceEvents_ resource was (and is) just using the default configuration defined up in the `relationshipBasedData` claim (which applies the "RelationshipsWithEdOrgsAndPeople" strategy). Naturally, if there's data defined on that resource it will be returned -- and in the sample data set there is, but without any authorization strategies applied to it, it doesn't contribute to the test coverage.

Additionally, the previous test actually covers all necessary functionality here -- that both ownership and relationship-based authorization are appropriately applied.